app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
In Python, imports are usually written in a specific order:
- Standard library imports
- Related third-party imports
- Local application/library specific imports

This is recommended by PEP8, the Python style guide.

Also, pls use `()` while importing multiple modules.
```suggestion
import datetime

from app.errors import (
    NotVaccinatedError,
    NotWearingMaskError, 
    OutdatedVaccineError, 
)
```